src/js/components/Carousel/Carousel.js

```diff
@@ -129,6 +133,8 @@ class Carousel extends Component {
 
     const NextIcon = theme.carousel.icons.next;
     const PreviousIcon = theme.carousel.icons.previous;
+    const NextIconDisabled = activeIndex >= lastIndex;
```

Comment:
I don't think these variables should start with upper case letters. We do that for NextIcon and PreviousIcon due to the JSX restriction on components having to behave this way. All other variables should start with lower case letters.